examples/places-autocomplete/src/autocomplete.rs

```diff
@@ -24,45 +24,27 @@ struct ImportPlacesOptions {
 
 #[derive(Default, Debug, Clone)]
 struct LegacyPlaceVisit {
-    id: i64,
     date: i64,
     visit_type: u8,
-    from_visit: i64,
 }
 
 #[derive(Default, Debug, Clone)]
 struct LegacyPlace {
     id: i64,
-    guid: String,
```

Comment:
there's not alot of value here, but I think this is reading a desktop places row, which possibly has *some* value? Maybe add a leading underscore on the names (and a comment about why?) I'm assuming this is what is described in https://github.com/rust-lang/rust/issues/81658, and that's one of the suggestions there?)

Reply:
I think I have a slight preference for commenting out fields like this rather than deleting it, adding the underscore, or adding the `#[allow(dead_code)]` annotation.  This seems like a nice way to indicate that the fields are present in the source data in case we want to use them some time in the future.  That said, I'm fine with any of the solutions.